lib/mapping/CMapInfo.h

```diff
@@ -61,6 +62,7 @@ class DLL_LINKAGE CMapInfo
 		h & campaign;
 		h & scenarioOptionsOfSave;
 		h & fileURI;
+		h & fullFileURI;

```

Comment:
@IvanSavenko Is it correct to add the var here?

Reply:
Yes, however this affects save format so please also:
- bump SERIALIZATION_VERSION by one (in CSerializer.h)
- add `if (version >= XXX)` check here (where XXX is new value of serialization version)

Reply:
The variable does not actually have to be in the savefile. Can I omit it then?